tests/testthat/test-get_datagrid.R

```diff
@@ -213,8 +213,92 @@ test_that("get_datagrid - models", {
   expect_identical(dim(get_datagrid(mod, include_random = FALSE, include_smooth = FALSE)), as.integer(c(10, 1)))
 })
 
+test_that("get_datagrid - emmeans", {
+  skip_if_not_installed("emmeans")
+  skip_on_cran()
+
+  data("mtcars")
+
+  mod1 <- glm(am ~ hp + factor(cyl), family = binomial("logit"), data = mtcars)
+  mod2 <- lm(mpg ~ hp + factor(cyl), data = mtcars)
+
+  hp_vals <- c(50, 100)
+
+  em1 <- emmeans::emmeans(mod1, ~ cyl | hp, at = list(hp = hp_vals))
+  em2 <- emmeans::regrid(em1)
+  em3 <- emmeans::emmeans(mod2, ~ cyl | hp, at = list(hp = hp_vals))
+
+  res <- get_datagrid(em1)
+  expect_equal(res, get_datagrid(em2))
```

Comment:
See lintrs. I suggest adding either (or both) `ignore_attr = TRUE` and optionally, `tolerance = ...`.

Reply:
Changed to `expect_identical()`